### ISSUE TITLE
Remove the "Import completed" popup

### ISSUE THREAD

**Issue by alexchandel** [sqlitebrowser/sqlitebrowser#1881]
Details for the issue
--------------------

#### What did you do?

Importing with "Table from CSV file..." results in an annoying popup after clicking "OK", once the import has finished.

#### What did you expect to see?

DB4S should simply return control to the GUI after importing is complete.

#### What did you see instead?

Although the still GUI locks up during import, after importing is complete, rather than simply returning control to the user, an annoying "Import completed" popup appears.

This popup serves no function whatsoever.  The user is already aware the import has completed, because the "Import CSV file" panel has disappeared, and the import has been logged.

This popup is merely one more thing for the user to click.  Please remove it.

Useful extra information
-------------------------

The info below often helps, please fill it out if you're able to. :)

#### What operating system are you using?

- [x] Mac OS: ( _version:10.14.4)

#### What is your DB4S version?

- [x] 3.11.2

#### Did you also

- [x] Try out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [x] Search for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by scottfurry** [comment]
Looking at [prj root]/src/ImportCsvDialog.cpp, `ImportCsvDialog::accept()` *should* show a dialog saying "Import completed" QMessageBox (line 218). Not sure if disabling this information feedback should be contemplated.

**Comment by justinclift** [comment]
Thinking it over... that dialog probably can be dropped completely without causing any user-awareness problems. :smile:

**Comment by scottfurry** [comment]
@justinclift  - or a status bar message "Import {Success|Failed}". Some minimally invasive visual indication of response to user action".

**Comment by justinclift** [comment]
Good thinking.  The status bar message approach instead sounds good. :smile:

**Comment by scottfurry** [comment]
Just to scope this out...
Do away with Information dialog feedback.
Feed success/fail to status bar.
Any/all extra messaging about import process gets dumped to internal logging(we have that, right?)

**Comment by justinclift** [comment]
Hmmm, we'd probably want a failure of the import to pop open a dialog, as it's the kind of thing that could be missed for some non-obvious failure scenarios.  eg partial imports maybe?

... and I'm not sure about our internal logging.  It's been mentioned we should have some kind of log file (not of the SQL kind) that could be enabled for debugging purposes, but we never seem to have gotten around to making one. :wink:

@MKleusberg @mgrojo That's correct re: the internal logging isn't it? (just checking) :smile:

**Comment by scottfurry** [comment]
....insert scope creep...

**Comment by justinclift** [comment]
Yeah, I think that's why the internal logging hasn't been done yet.  Potential can of worms thing. :smile:

**Comment by alexchandel** [comment]
Simple enough:  show that popup if any import error occurs.  Silently return control to the main GUI if it succeeds.

**Comment by scottfurry** [comment]
I thing to note...
Dialog popup is removed. However, the notion I had earlier to display information to the Status Bar is not implemented at this time. That's a something that is little more complicated due to overall DB4S class structure. Implementation requires a little more planning.
